mysql-test/suite/rocksdb/t/ttl_primary_with_partitions.test

```diff
@@ -5,8 +5,6 @@ DROP TABLE IF EXISTS t1;
 DROP TABLE IF EXISTS t2;
```

Comment:
Needs have_debug.inc and same comment re. release/debug build testing as for ttl_primary.test